selfdrive/car/toyota/toyotacan.py

```diff
@@ -35,7 +35,7 @@ def create_accel_command(packer, accel, pcm_cancel, standstill_req, lead, acc_ty
     "ACC_TYPE": acc_type,
     "DISTANCE": 0,
     "MINI_CAR": lead,
-    "PERMIT_BRAKING": 1,
+    "PERMIT_BRAKING": 1 if accel <= 0 else 0,
```

Comment:
@HaraldSchafer this might not be the best way to do this correctly. If you're on a downward facing hill for example, 0 or even some small positive acceleration means we might want to brake. Should we use the localizer to find the coasting acceleration here?

Reply:
Can you test it on a hill and look at the plots? Maybe they deal with this internally. Just want to be sure they don't before we start writing logic.

Reply:
They do not, it fails to brake no matter how sharp the decline is